docarray/math/evaluation.py

```diff
@@ -12,20 +12,25 @@ def _check_k(k):
 
 
 def r_precision(binary_relevance: List[int], **kwargs) -> float:
```

Comment:
there needs to be some tests testing the new behavior

Reply:
Yes I agree. Currently there are no tests for the metric functions. I have created a test which tests the behavior together with the evaluate function functions in #617 (which has to be adopted -after one of those Pars is merged), but I can also create tests which directly test the functions docarray.math here.

Reply:
add unit tests yes